client/packages/common/src/hooks/useNativeClient/helpers.ts

```diff
@@ -0,0 +1,56 @@
+import { Capacitor, registerPlugin } from '@capacitor/core';
+import { FrontEndHost, NativeMode, NativeAPI } from './types';
+
+export const setPreference = async (
```

Comment:
the preference methods are new - the rest of this is just moved from `useNativeClient`